src/containers/cats/index.js

```diff
@@ -1,13 +1,13 @@
 import React, { Component } from 'react'
 import { connect } from 'react-redux'
-import { onCat } from 'redux/actions/cats'
+import { onCat, onCatAdopt } from 'redux/actions/cats'
```

Comment:
You've changed the name of this action now, so it needs changing here as well (your console should be giving you an error on this one)